sympy/series/limits.py

```diff
@@ -163,6 +163,9 @@ def doit(self, **hints):
 
         e, z, z0, dir = self.args
 
+        if z0 is S.ComplexInfinity:
+            raise NotImplementedError()
```

Comment:
There should be some text in NotImplementedError, otherwise how will the user know that the problem is with zoo and not with something else in the formula (like, in the expression `e`)?  For example, `NotImplementedError("Limits at complex infinity are not implemented")`